android/sdl_android/src/androidTest/java/com/smartdevicelink/test/rpc/requests/ShowConstantTbtTests.java

```diff
@@ -71,8 +71,8 @@ protected JSONObject getExpectedParameters(int sdlVersion) {
             result.put(ShowConstantTbt.KEY_SOFT_BUTTONS, TestValues.JSON_SOFTBUTTONS);
             result.put(ShowConstantTbt.KEY_ETA, TestValues.GENERAL_STRING);
             result.put(ShowConstantTbt.KEY_MANEUVER_COMPLETE, true);
-            result.put(ShowConstantTbt.KEY_MANEUVER_DISTANCE, TestValues.GENERAL_DOUBLE);
-            result.put(ShowConstantTbt.KEY_MANEUVER_DISTANCE_SCALE, TestValues.GENERAL_DOUBLE);
+            result.put(ShowConstantTbt.KEY_DISTANCE_TO_MANEUVER, TestValues.GENERAL_DOUBLE);
+            result.put(ShowConstantTbt.KEY_DISTANCE_TO_MANEUVER_SCALE, TestValues.GENERAL_DOUBLE);
```

Comment:
@kostyaBoss On line 75 I would rename `KEY_DISTANCE_TO_MANEUVER_SCALE` back to its old variable name `KEY_MANEUVER_DISTANCE_SCALE`.  You can apply this change throughout the class.